masterlist.yaml

```
@@ -7915,6 +7915,48 @@ plugins:
   - name: 'Improved dungeons.esp'
     msg: [ *obsolete ]
 
+  - name: '(Mage Equipment|Francesco''s Mage Equipment)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/18875/'
+        name: 'Mage Equipment 21'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Mage Equipment 21 ESP' ]
+        condition: 'many("(Mage Equipment|Francesco''s Mage Equipment)\.esp")'
+    tag:
+      - Actors.DeathItem
+      - Delev
+      - Enchantments
+      - Graphics
+      - Invent.Add
+      - Invent.Remove
+      - Scripts
+      - Stats
+      - Text
+  - name: 'Mage Equipment.esp'
+    msg:
+      - <<: *obsolete
+        subs: [ '[Mage Equipment 21](https://www.nexusmods.com/oblivion/mods/18875/)' ]
+        condition: 'checksum("Mage Equipment.esp", 27FA1E0F) or checksum("Mage Equipment.esp", 6693E84F)'
+      - <<: *useVersion
+        subs: [ 'Francesco''s Mage Equipment.esp' ]
```

Adjusting the sub to fit the message better.

```suggestion
        subs: [ 'Francesco''s Creatures and Items Mod' ]
```